app/templates/home.html

```diff
@@ -1,11 +1,110 @@
 {% extends "base.html" %}
 
 {% block content %}
+<br>
+<h2>Welcome Back! {{current_user.firstname}} :)</h2>
 
+<style>
+  .button {
```

Comment:
should organize styling by putting in a separate css file, considering a doc for website specifying colors, sizes, typefaces etc.